### DISCUSSION TITLE
Replacing Figlet with Spectre.Console

### DISCUSSION THREAD

**Discussion by huw-dhcw** [cake-build/cake#4097] [Q&A] [answered]
I've been using Cake.Figlet :
`Information(new FigletText("My Project"));
`

but this only supports .net 5.0.

[https://github.com/cake-contrib/Cake.Figlet/issues/38](https://github.com/cake-contrib/Cake.Figlet/issues/38) states:

> the goal of this addin is to add FIGlet font support to Cake. Since we're now shipping [Spectre.Console](https://github.com/spectresystems/spectre.console) as part of Cake with 1.0, Cake comes with out-of-the-box FIGlet support:
> 
```
 AnsiConsole.Render(
     new FigletText("Hello")
         .LeftAligned()
         .Color(Color.Red));

```
[docs](https://cakebuild.net/docs/writing-builds/console-output#advanced-output) also indicate that Spectre is the way to go, but I can't find any working examples. I've tried the AnsiConsole command from above but I get a compilation error (as I don't have a using statement)

Looks like we [don't have an alias](https://github.com/cake-build/cake/issues/3135) for it, so how do I use it in my script?

**Comment by patriksvensson** [comment, marked as answer]
You should be able to use the code above by adding "using Spectre.Console" to your script.
